serverless/networking/serverless-ossm-jwt.adoc

```diff
@@ -16,21 +16,19 @@ You can enable JSON Web Token (JWT) authentication for Knative services.
 
 [IMPORTANT]
 ====
-Adding sidecar injection to Pods in system namespaces such as `knative-serving` and `knative-serving-ingress` is not supported.
+Adding sidecar injection to pods in system namespaces such as `knative-serving` and `knative-serving-ingress` is not supported.
 ====
 
 .Procedure
 
 . Create a policy in your serverless application namespace that only allows requests with valid JSON Web Tokens (JWT):
```

Comment:
Wondering if this counts as a "single step procedure" (needing an `*` instead of a `.`) even though it has multiple substeps. Or maybe it could be re-written slightly as just two regular steps?

Reply:
Changed this slightly to make the first line part of the intro and just call out the two steps, lmk what you think :)

Reply:
Perfecto!